### ISSUE TITLE
Docs: Fix typos in the dashboard-app Learn course

### ISSUE THREAD

**Issue by jung0han** [vercel/next.js#57696]
### What is the improvement or update you wish to see?

I would like to request the correction of typo in the dashboard-app course document.

### Is there any context that might help us understand?

I created the CSS file inside the UI folder, but when I import it from the `page.tsx` file under the `app `folder.
It fails to load because I imported it with `import styles from './home.module.css';` the css file is not imported.

Contents of the document
```js
import styles from './home.module.css';
 
//...
<div className={styles.shape}></div>;
```

Route information that needs to be changed
```js
import styles from './ui/home.module.css';
```

Please make the necessary corrections to the typos to enhance the clarity of the document. We kindly request that you review and address these corrections. Thank you.

### Does the docs page already exist? Please link to it.

https://nextjs.org/learn/dashboard-app/css-styling#css-modules

**Comment by balaji-sivasakthi** [comment]
@jung0han I will fix it.

**Comment by balaji-sivasakthi** [comment]
@jung0han I will fix it.

**Comment by Codensity30** [comment]
@jung0han Yes, I also have found various typos or file location mismatches in this specific course. How can we report this and can we fix it?

**Comment by mukham12** [comment]
I also couldn’t find the right place to report such issues so I have created issue #57689.

**Comment by balaji-sivasakthi** [comment]
> I also couldn’t find the right place to report such issues so I have created issue #57689.

The Learn module is not an open source site.

**Comment by balaji-sivasakthi** [comment]
@mukham12 @mukham12 the issue should be taken care of nextjs core team.

**Comment by mukham12** [comment]
@balaji-sivasakthi Agreed. That is why I created the above mentioned issue with the typos and errors I caught so far in the first twelve chapters.

**Comment by mukham12** [comment]
@jung0han,

I believe it would be more appropriate to move this issue to the [next-learn](https://github.com/vercel/next-learn) repository since that is where the code for this is written. In fact, I already have an open issue there. If you would like to help keep things organized, you can comment about this matter under my [existing issue.](https://github.com/vercel/next-learn/issues/336)

**Comment by mukham12** [comment]
@jung0han,

This appears to have been fixed. Could you please check and close the issue if that is indeed the case?
